cob_light/common/include/modeExecutor.h

```diff
@@ -67,26 +68,31 @@ class ModeExecutor
 	ModeExecutor(IColorO* colorO);
 	~ModeExecutor();
 
-	void execute(Mode* mode);
-	void execute(cob_light::LightMode requestMode);
+	uint64_t execute(boost::shared_ptr<Mode> mode);
+	uint64_t execute(cob_light::LightMode requestMode);
 
 	int getExecutingPriority();
 	int getExecutingMode();
+	uint64_t getExecutingUId();
 
+	void pause();
+	void resume();
```

Comment:
`pause` and `resume` is something the light driver handles internally only, right?
Depending on the modes that have been started and their respective priority...
It is not exposed to the user, correct?